scene/animation/animation_player.h

```diff
@@ -94,10 +94,10 @@ class AnimationPlayer : public AnimationMixer {
 			return from == bk.from && to == bk.to;
 		}
 		bool operator<(const BlendKey &bk) const {
-			if (from == bk.from) {
-				return to < bk.to;
+			if (String(from) == String(bk.from)) {
```

Comment:
Why this? There's no reason to compare them as strings, that's significantly slower

Reply:
Good point and thanks for the review, I will change this back to `if (from == bk.from)`. Testing it locally seems to work fine and not result in sort variations.

Do you know if comparing two `StringName`s  that have the same string literal will always point to the same mem address?

Reply:
You should just use `StringName::AlphCompare` here, like so:
```cpp
return StringName::AlphCompare()(from, bk.from) < 0;
```